index.html

```diff
@@ -622,11 +622,13 @@ <h1>Streaming GCN<br>Notices in Python</h1>
 
     <section>
       <div class="image-and-caption">
-        <img src="img/screen-shot-07-step-4.png" width="900" alt="Screen shot">
+        <img src="img/screen-shot-07-step-4.png" width="850" alt="Screen shot">
         <div data-markdown>
           <textarea data-template>
             ## Step 4:<br>Get Sample Code
-            Copy and paste Python client code or download it to your computer to run.
+            Copy and paste Python client code or download it to your computer to run. <br>
+            <br>
```

Comment:
This is Markdown. Create paragraphs by skipping a line.

```suggestion
            Copy and paste Python client code or download it to your computer to run.

```